tests/Unit/DataStructures/DataBox/Test_DataBoxPrefixes.cpp

```diff
@@ -6,22 +6,55 @@
 #include <string>
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
-#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Utilities/TMPL.hpp"
+
+class DataVector;
+// IWYU pragma: no_forward_declare Tags::Flux
 
 namespace {
 struct Tag : db::SimpleTag {
-  static std::string name() noexcept { return "Tag"; }
   using type = double;
 };
+
+struct TensorTag : db::SimpleTag {
+  using type = tnsr::I<DataVector, 2>;
+};
 }  // namespace
 
 SPECTRE_TEST_CASE("Unit.DataStructures.DataBox.Prefixes",
                   "[Unit][DataStructures]") {
+  /// [dt_name]
+  CHECK(db::tag_name<Tags::dt<Tag>>() == "dt(" + db::tag_name<Tag>() + ")");
+  /// [dt_name]
+  using Dim = tmpl::size_t<2>;
+  using Frame = Frame::Inertial;
+  using VariablesTag = Tags::Variables<tmpl::list<TensorTag>>;
+  /// [flux_name]
+  CHECK(db::tag_name<Tags::Flux<TensorTag, Dim, Frame>>() ==
+        "Flux(" + db::tag_name<TensorTag>() + ")");
+  CHECK(db::tag_name<Tags::Flux<VariablesTag, Dim, Frame>>() ==
+        "Flux(" + db::tag_name<VariablesTag>() + ")");
+  /// [flux_name]
```

Comment:
Are we doing these snippets now to print the `name` to docs? If the idea is to do this for all tags we should add something to our guidelines

Reply:
This is just something I've been doing.  I don't think there has been any wider discussion of it.

I do it because I consider the name tests to be pointless otherwise, as they are just copy-pastes of the source code.  This way even if both versions are wrong we know the documentation describes the actual behavior.

Reply:
Yes I like it 👍